tests/neuro_san/client/assert_forwarder.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+from typing import Any
+
+
+class AssertForwarder:
+    """
+    Interface definition for forwarding asserts to different test infrastructure systems.
+    """
```

Comment:
A interface that allows the AgentEvaluators to assert things about what they are testing without necessarily hooking into the python UnitTest infrastructure and/or actually stopping test operation.